Qonversion/Qonversion/NetworkLayer/Error/ErrorConstants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ErrorConstants.swift
+//  Qonversion
+//
+//  Created by Suren Sarkisyan on 19.02.2024.
+//
+
+import Foundation
+
+enum ErrorCodes: Int {
+    case unauthorized = 401
+    case paymentRequired = 402
+    case forbidden = 403
+    case internalErrorStart = 500
+    case internalErrorEnd = 599
```

Comment:
Doesn't these names suit more?
```suggestion
    case internalErrorMin = 500
    case internalErrorMax = 599
```